azure-communication-ui/build.gradle

```diff
@@ -45,6 +45,7 @@ buildscript {
         shouldNotCheckTaskRoot = {
             return rootProject.hasProperty("disableTaskRootCheck") && rootProject.getProperty("disableTaskRootCheck")
         }
+        compose_version = '1.1.0-beta01'
```

Comment:
Please do not add compose version. Add Fluent Ui button dependency, it will pull compose and we will use same version